projects/pgai/db/tests/vectorizer/test_named_vectorizer.py

```diff
@@ -51,9 +37,50 @@ def test_named_vectorizer():
                 , ('how to make stir fry', '2022-01-06'::timestamptz, 'pick up the phone and order takeout')
             """)
 
-            # drop the drop_me column
-            cur.execute("alter table website.blog drop column drop_me")
 
+@pytest.mark.skipif(os.getenv("PG_MAJOR") == "15", reason="does not support pg15")
```

Comment:
why does this not work on pg15?

Reply:
Not sure if more errors will raise besides that one